tests/test_api_v1.py

```diff
@@ -493,8 +495,18 @@ def test_set_state(mock_rmtree, mock_exists, state, app, client, db, worker_auth
     assert fetched_request['state_history'][0]['state_reason'] == state_reason
     assert fetched_request['state_history'][0]['updated']
     assert fetched_request['state_history'][1]['state'] == 'in_progress'
-    mock_exists.assert_called_once_with('/tmp/cachito-archives/bundles/temp/1')
-    mock_rmtree.assert_called_once_with('/tmp/cachito-archives/bundles/temp/1')
+    mock_exists.assert_called_once_with(
+        os.path.join(
+            tempfile.gettempdir(),
+            'cachito-archives/bundles/temp/1'
```

Comment:
Adding a trailing comma is a good practice.
You can read https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas to see why.

```suggestion
            'cachito-archives/bundles/temp/1',
```

Reply:
thanks @mprahl - makes sense. 
perhaps we could add these style / formatting / static code analysis checks to CI with something like codeclimate, etc?